src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -66,13 +65,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     long itemId;
+    
+    System.out.println("TEST:" + request.getParameter("itemId"));
```

Comment:
I think you should remove this testing print statements from the PR since they shouldn't be part of master and the production code